cypress/integration/load/LargeFiler.spec.js

```diff
@@ -0,0 +1,166 @@
+import { isBeta, isCI } from '../../support/helpers'
+import { getDefaultConfig } from '../../../src/common/configUtils'
+
+const {
+  HOST,
+  USERNAME,
+  PASSWORD,
+  INSTITUTION,
+  ACTION_DELAY,
+  TEST_DELAY,
+  ENVIRONMENT,
+  AUTH_BASE_URL,
+  AUTH_REALM,
+  AUTH_CLIENT_ID,
+  YEARS,
+} = Cypress.env()
+
+const years = [2020]
+const timeout3hours = 10800000
```

Comment:
think it'd be easier to read if we did `3 * 60 * 60 * 1000`, so if we wanted to change anything later, it'd be more easily understandable.

Reply:
I like it 👍🏽